README.md

```diff
@@ -48,7 +50,34 @@
 6. Select all datapoints and add them to a new dataset
 7. From http://localhost:4004/data-lake, Run the embedding analysis
 
+**Windows:**
+
+1. Clone this repo
+    ```bash
+    git clone --recurse-submodules git@github.com:dioptra-ai/katiml.git
+    ```
 
+2. Start all services with docker-compose
+    ```bash
+    cd katiml
+    type nul > .env
+    docker compose up --build
+    ```
+3. If you're starting for the first time, run the schema migration
+    ```bash
+    cd services/ingestion/schemas/pgsql
```

Comment:
not a windows expert but aren't path using `\` ?

Reply:
yes, but it also handles / in the same way